app/views/posts/show.html.erb

```diff
@@ -1,7 +1,22 @@
-<h1>
-    <%= @post.title %>
-</h1>
+<h1><%= "Post #{params[:index].to_i + 1} by #{@post.author.name}" %></h1>
+
 
 <p>
-    <%= @post.text %>
-</p>
\ No newline at end of file
+  Comments: <%= @post.comments_counter %>,
+  Likes: <%= @post.likes_counter %>
+</p>
+<p><%= @post.text %></p>
+
+<div>
+  <% if @post.comments.present? %>
+    <ul>
+      <% @post.comments.each do |comment| %>
+        <li>
+          <strong><%= comment.user.name %>:</strong> <%= comment.text %>
+        </li>
```

Comment:
- [ ] Again here kindly ensure you follow the given frame in terms of layout of items on the page so as to meet the requirements

Your design
![image](https://github.com/Eccb7/Pi-Blogs/assets/81932478/6b102786-183b-4243-b6bf-01849723717b)


Given design
![image](https://github.com/Eccb7/Pi-Blogs/assets/81932478/e122d788-ed0b-4965-a9fc-59891ed6f36e)